### ISSUE TITLE
Results of BERT SRL model on CoNLL-2005 dataset

### ISSUE THREAD

**Issue by swarnaHub** [allenai/allennlp#3395]
I was wondering if your BERT SRL model achieves state-of-the-art results on CoNLL-2005 dataset too.  I implemented a similar model (with just a classification layer and Viterbi decoding) on top of the Hugging Face Transfomers repository but my results are not that great. If you have tested your model on the CoNLL-2005 dataset, may be you can share some insights. @nelson-liu Tagging you, since some of the latest commits on BERT SRL were by you. Looking forward to hearing from you soon.

**Comment by DeNeutoy** [comment]
Hi!

Sorry but we haven't evaluated on the CoNLL-2005 dataset, i've only ever used the 2012 one. I don't actually have access to it 😅 

However, results on the two datasets are pretty correlated, so i'd expect it to be similar in performance.

**Comment by swarnaHub** [comment]
Thanks for your quick response. My understanding was also that the numbers should not differ much between the datasets. However, I do not have access to the OntoNotes dataset and hence can't verify the numbers. 

I looked at your training configuration and it seems that you train the model for 15 epochs. Mine mostly saturates in 3 epochs. If you have some training logs/remember how the training proceeded, I would love to hear more. Thanks!

**Comment by DeNeutoy** [comment]
[training_run.tar.gz](https://github.com/allenai/allennlp/files/3768809/training_run.tar.gz)

Training run attached - note that this is not the exact same run as the model in the demo (only for 8 epochs) but you should get an idea.

Certainly fine tuning does saturate pretty early and there is not much variation between different numbers of epochs when using a slanted triangular learning rate.

One thing to note - this model was trained before #667, so the metrics in the logs *track slightly below* what they actually are. So the final number reported in the demo used a script which actually evaluated with the official perl script, which now happpens automatically. So if you train a model again with allennlp, it will look slightly better - but actually this is just a slight change in how the metric is tracked.  

If you're interested in getting a copy of ontonotes 5.0 feel free to email me ...

**Comment by swarnaHub** [comment]
Mark, I have emailed you. Would be great if you can respond on the email thread. Thanks again!
